task/power/src/bsp/cosmo_a.rs

```diff
@@ -0,0 +1,343 @@
+    rail_controller!(Core, raa229620A, vddcr_cpu1_a0, A0),
+    rail_controller!(Core, raa229620A, vddio_sp5_a0, A0),
+    rail_controller_notemp!(Core, isl68224, v1p1_sp5_a0, A0),
+    rail_controller_notemp!(Core, isl68224, v1p8_sp5_a1, A0), // XXX A0 or A2?
+    rail_controller_notemp!(Core, isl68224, v3p3_sp5_a1, A0), // XXX A0 or A2?
+    rail_controller!(Sys, tps546B24A, v3p3_sp_a2, A2),
+    rail_controller!(Sys, tps546B24A, v5_sys_a2, A2),
+    rail_controller!(Sys, tps546B24A, v1p8_sys_a2, A2),
+    rail_controller!(Sys, tps546B24A, v0p96_nic_vdd_a0hp, A0),
+    adm1272_controller!(HotSwap, v54p5_ibc_a3, A2, Ohms(0.000075)),
+    lm5066_controller!(
+        Fan,
+        v54p5_fan_east,
+        A2,
+        Ohms(0.007),
+        drv_i2c_devices::lm5066::CurrentLimitStrap::VDD
+    ),
+    lm5066_controller!(
+        Fan,
+        v54p5_fan_central,
+        A2,
+        Ohms(0.007),
+        drv_i2c_devices::lm5066::CurrentLimitStrap::VDD
+    ),
+    lm5066_controller!(
+        Fan,
+        v54p5_fan_west,
+        A2,
+        Ohms(0.007),
+        drv_i2c_devices::lm5066::CurrentLimitStrap::VDD
+    ),
+    max5970_controller!(HotSwapIO, v3p3_m2a_a0hp, A0, Ohms(0.003)),
+    max5970_controller!(HotSwapIO, v3p3_m2b_a0hp, A0, Ohms(0.003)),
+    max5970_controller!(HotSwapIO, v12_u2a_a0, A0, Ohms(0.005)),
+    max5970_controller!(HotSwapIO, v3p3_u2a_a0, A0, Ohms(0.008)),
+    max5970_controller!(HotSwapIO, v12_u2b_a0, A0, Ohms(0.005)),
+    max5970_controller!(HotSwapIO, v3p3_u2b_a0, A0, Ohms(0.008)),
+    max5970_controller!(HotSwapIO, v12_u2c_a0, A0, Ohms(0.005)),
+    max5970_controller!(HotSwapIO, v3p3_u2c_a0, A0, Ohms(0.008)),
+    max5970_controller!(HotSwapIO, v12_u2d_a0, A0, Ohms(0.005)),
+    max5970_controller!(HotSwapIO, v3p3_u2d_a0, A0, Ohms(0.008)),
+    max5970_controller!(HotSwapIO, v12_u2e_a0, A0, Ohms(0.005)),
+    max5970_controller!(HotSwapIO, v3p3_u2e_a0, A0, Ohms(0.008)),
+    max5970_controller!(HotSwapIO, v12_u2f_a0, A0, Ohms(0.005)),
+    max5970_controller!(HotSwapIO, v3p3_u2f_a0, A0, Ohms(0.008)),
+    max5970_controller!(HotSwapIO, v12_u2g_a0, A0, Ohms(0.005)),
+    max5970_controller!(HotSwapIO, v3p3_u2g_a0, A0, Ohms(0.008)),
+    max5970_controller!(HotSwapIO, v12_u2h_a0, A0, Ohms(0.005)),
+    max5970_controller!(HotSwapIO, v3p3_u2h_a0, A0, Ohms(0.008)),
+    max5970_controller!(HotSwapIO, v12_u2i_a0, A0, Ohms(0.005)),
+    max5970_controller!(HotSwapIO, v3p3_u2i_a0, A0, Ohms(0.008)),
+    max5970_controller!(HotSwapIO, v12_u2j_a0, A0, Ohms(0.005)),
+    max5970_controller!(HotSwapIO, v3p3_u2j_a0, A0, Ohms(0.008)),
+    ltc4282_controller!(HotSwapIO, v12_ddr5_abcdef_a0, A0, Ohms(0.001)),
+    ltc4282_controller!(HotSwapIO, v12_ddr5_ghijkl_a0, A0, Ohms(0.001)),
+    max5970_controller!(HotSwapIO, v12p0_nic_a0hp, A0, Ohms(0.003)),
+    max5970_controller!(HotSwapIO, v5p0_nic_a0hp, A0, Ohms(0.003)),
+    // XXX Gimlet doesn't include sharkfin HSCs here.  Why not?
```

Comment:
I don't think I understand that comment; I think Gimlet does include the Sharkfin HSCs?